convert2rhel/checks.py

```diff
@@ -791,10 +791,20 @@ def is_loaded_kernel_latest():
         # Repoquery failed to detected any kernel or kernel-core packages in it's repositories
         # we allow the user to provide a environment variable to override the functionality and proceed
         # with the conversion, otherwise, we just throw an critical logging to them.
-        unsupported_skip = os.environ.get("CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK", "0")
-        if unsupported_skip == "1":
+        allow_older_envvar_names = (
+            "CONVERT2RHEL_ALLOW_SKIP_KERNEL_CURRENCY_CHECK",
+            "CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK",
+        )
+        if any(envvar in os.environ for envvar in allow_older_envvar_names):
+            if "CONVERT2RHEL_ALLOW_SKIP_KERNEL_CURRENCY_CHECK" not in os.environ:
```

Comment:
I would check for the old name instead of the new one... It seems more correct to me, as the message that we are printing is regarding using the deprecated name instead of the new one.

```suggestion
            if "CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK" not in os.environ:
```

Reply:
Oh, just now I saw that @abadger commented something on #683 that also affects this one, so I will just close this comment right here.